bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTO.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2016 Markus Rathgeb
```

Comment:
As discussed in the past, the agreed procedure in this project is [A.2 from the guidelines](https://www.eclipse.org/smarthome/documentation/development/guidelines.html) and keeping track of the copyrights through A.4 (author tags). I would actually like to change the automatically generated headers to be [similar to this](https://github.com/openhab/openhab2-addons/blob/master/src/etc/header.txt#L1). Wdyt?